source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -282,6 +282,107 @@ the Query API:
      }
    ]
 
+Function Operations
+~~~~~~~~~~~~~~~~~~~
+
+You can apply custom functions to values of type
+``MqlValue`` using the methods described in this section.
+
+To improve readability and brevity in your code, you can move redundant code into
```

Comment:
```suggestion
To improve readability and allow for code reuse, you can move redundant code into
```
Maybe brevity isn't worth mentioning explicitly (the corresponding MQL is relatively short by comparison, in part because we don't have `average` implemented yet)